Userland/Utilities/md.cpp

```diff
@@ -56,7 +56,12 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         return 1;
     }
 
-    DeprecatedString res = html ? document->render_to_html() : document->render_for_terminal(view_width);
-    out("{}", res);
+    if (html) {
+        out("{}", document->render_to_html());
+    } else {
+        auto res = TRY(document->render_for_terminal(view_width));
+        out("{}", res);
```

Comment:
Is there a reason why we are using a temporary variable here instead of passing the string to `out` directly (like in the upper branch)?

(This comment was supposed to accompany my "approve" review.)

Reply:
There is no particular reason, I just was not aware that you could do that with a statement wrapped in `TRY()`.

I have another PR that is in draft (https://github.com/SerenityOS/serenity/pull/19385/files) and I could tidy this up there if you would like?

Reply:
Update: I have now changed the follow-up PR from a draft to ready-for-review, and I have also updated that PR to remove the temporary variable

Reply:
Fine by me if someone signs off on it, but I'd personally prefer if we just never introduce that variable to begin with. :^)

Reply:
I have gone ahead and changed it in this pull request.

RE: "someone signs off on it", who is the someone that I would need to sign off/merge this pull request? I had assumed anyone who is a member of the SerenityOS organisation would have merge rights, but I'm wondering if that is correct?

Reply:
Only the current 10 maintainers have merge/push rights. The other members of the SerenityOS GitHub organization only manage issues and tag pull requests.

Reply:
Okay, cool thanks for letting me know :)

Is there a document that explains this process that I might have glossed over? My guess would be that once the label is changed from "pr needs review" to something else then that would alert a maintainer to take a look at the PR and merge it if they are happy with it?